sdks/java/io/kafka/pom.xml

```diff
@@ -139,6 +139,12 @@
       <artifactId>auto-value</artifactId>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-expression</artifactId>
+      <version>4.3.5.RELEASE</version>
```

Comment:
We'd like to keep all versioning information at the top-level; just like other dependencies.

License: OK.